token/client.go

```diff
@@ -11,6 +11,7 @@ import (
 const (
 	Card stripe.TokenType = "card"
 	Bank stripe.TokenType = "bank_account"
+	PII	 stripe.TokenType = "pii"
```

Comment:
@gukjoon Can you run any Go files you touched through a `gofmt -w <file>`? It'll stripe little whitespace discrepancies like this out one out. Thanks!